components/cluster/pacemaker/manifests/sample-manifest.p5m

```diff
@@ -134,6 +134,29 @@ file path=usr/lib/python2.7/site-packages/cts/logging.py
 file path=usr/lib/python2.7/site-packages/cts/patterns.py
 file path=usr/lib/python2.7/site-packages/cts/remote.py
 file path=usr/lib/python2.7/site-packages/cts/watcher.py
+file path=usr/sbin/$(MACH64)/attrd_updater
```

Comment:
Is the manifest-sample.p5m correct? Binaries shouldn't move out of /usr/bin.

Reply:
corosync and pacemaker use mainly /usr/sbin.
And tools loke crm and pcs search the binaries in this paths.

Reply:
He speaks about $(MACH64). Regenerate sample-manifest, please.